src/services/ui/src/components/Inputs/button.tsx

```diff
@@ -13,8 +13,7 @@ const buttonVariants = cva(
           "bg-primary text-slate-50 hover:bg-accent hover:text-accent-foreground",
         destructive:
           "bg-destructive text-destructive-foreground hover:bg-destructive/90",
-        outline:
-          "border border-input bg-background hover:bg-accent hover:text-accent-foreground",
+        outline: "border-2 border-primary text-primary font-bold",
```

Comment:
do we not want any sort of hover state on this button?

Reply:
@benjaminpaige I didn't see a hover state mentioned in Figma unless I missed it. Do you have a specific hover state in mind?

Reply:
Good idea. I restored the hover state we'd already had for the outline variation. 👍🏻